src/Scoringline/SendinblueApi/Exception/InvalidFileException.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+/**
+ * This file is a part of scoringline sendinblue api package
+ *
+ * (c) Scoringline <m.veber@scoringline.com>
+ *
+ * @author Joni Rajput <joni@sendinblue.com>
```

Comment:
the `@author` tag should be on the class comment, not global file comment (same notice for every files)

Look on symfony classes :-)